meta-schema.graphql

```diff
@@ -0,0 +1,115 @@
+# Base types
+type Party {
+	id:ID!
+	idType:String! #PartyId, IPI, ISNI, MBID
+	displayName:String!
+	firstName: String
+	lastName: String
+}
+
+type Organisation {
+	id:String!
+	idType:String! #DPid, IPI, MBID
+	displayName:String!
+	formalName:String
+}
+
+# Music Types
+type TerritoryAgreement {
+	territoryCode:Int! #CISAC Territory standard
+	condition:String! # CWR inclusion/exclusion indicator
+}
+
+type InterestedPartyAgreement {
+	agreementRoleCode:String!
+	interestedPartyNumber:Int!
+	ipi:Int
+	ipiBase:Int
+	lastName:String!
+	firstName:String
+	prSociety:String
+	prShare:Int
+	mrSociety:String
+	mrShare:Int
+}
+
+type MusicWork {
+	iswc:String
+	workTitle:String!
+	territories:[TerritoryAgreement]
+	interestedParty:[InterestedPartyAgreement]
+	publishers:[Organisation]
+	composers:[MusicComposer]
+	recordings:[MusicRecording]
+}
+
+# DDEX:ERN:SoundRecording
+type MusicRecording {
+	isrc: String
+	iswc: String
+	displayTitle:String!
+	displayArtist:String!
```

Comment:
Should this not be a `Party`?

Reply:
No, but it should be an array. The `contributors` field covers the referenced `Party`s. This field is for display purposes, ie:
```
displayArtist: "Wu-Tang"
contributors: "Inspectah Deck", "RZA", "GZA", "ODB", "etc etc"
```